README.md

```diff
@@ -885,6 +887,47 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may encounter a `build failed` error when installing Python through pyenv. 
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to ensure that all requirements for the Python build environment from the pyenv wiki 
```

Comment:
What happened to the link to the wiki? See the previous 2 comments. https://github.com/DataBiosphere/azul/pull/3311#discussion_r696893231